portal-user-service/src/main/resources/portaluser-service-openapi.yaml

```diff
@@ -311,6 +311,32 @@ paths:
         500:
           $ref: '#/components/responses/GenericApiError'
           
+  /api/portalUser/customerIdsForUsername:
```

Comment:
same comments as fo rthe other yaml file - path and return type should match the values in PortalUserController